dali/operators/reader/loader/video/frames_decoder.h

```diff
@@ -90,107 +90,114 @@ struct MemoryVideoFile {
 
 /**
  * @brief Object representing a video file. Allows access to frames and seeking.
- * 
+ *
  */
 class DLL_PUBLIC FramesDecoder {
  public:
   static const std::vector<AVCodecID> SupportedCodecs;
 
   /**
    * @brief Construct a new FramesDecoder object.
-   * 
+   *
    * @param filename Path to a video file.
    */
   explicit FramesDecoder(const std::string &filename);
 
 
   /**
    * @brief Construct a new FramesDecoder object.
-   * 
+   *
    * @param memory_file Pointer to memory with video file data.
    * @param memory_file_size Size of memory_file in bytes.
    * @param build_index If set to false index will not be build and some features are unavailable.
-   * 
+   *
    * @note This constructor assumes that the `memory_file` and
    * `memory_file_size` arguments cover the entire video file, including the header.
    */
   FramesDecoder(const char *memory_file, int memory_file_size, bool build_index = true);
 
   /**
    * @brief Number of frames in the video. It returns 0, if this information is unavailable.
-   * 
+   *
```

Comment:
When you touch these lines - they should just go away if we don't have anything after "brief".

Reply:
The IDE trimmed the trailing whitespaces. Nevertheless, removed.